addons/shape-categories/addon.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "Old Shaped Categories",

```

Comment:
Minor detail, all of the other addons (with a few exceptions) don't start every word with an uppercase letter. They actually look like this:
```suggestion
    "name": "Old shaped categories",
```

Reply:
Done with changes